### PR TITLE
feat(git): Add `mini.git`

### PR DESCRIPTION
**Pull request by Uzaaft**

### CODE REVIEW

**Comment by Uzaaft** [comment]
I don't think opts is needed for this plugin.

**Comment by mehalter** [comment]
it's not necessary but there are `opts` and a setup call, so might as well put it 🤷🏼

File: lua/astrocommunity/git/mini-git/init.lua [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---@type LazySpec
+return {
+  "echasnovski/mini-git",
+  event = "User AstroFile",
```
Comment (mehalter) [inline lua/astrocommunity/git/mini-git/init.lua:4]:
```suggestion
  event = "User AstroGitFile",
```

only load automatically for git files

File: lua/astrocommunity/git/mini-git/init.lua [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  cmd = { "Git" },
+  version = false,
+  main = "mini.git",
```
Comment (mehalter) [inline lua/astrocommunity/git/mini-git/init.lua:7]:
```suggestion
```

this is automatic by lazy i'm pretty sure

File: lua/astrocommunity/git/mini-git/init.lua [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---@type LazySpec
+return {
+  "echasnovski/mini-git",
+  event = "User AstroFile",
+  cmd = { "Git" },
+  version = false,
```
Comment (mehalter) [inline lua/astrocommunity/git/mini-git/init.lua:6]:
```suggestion
```

we don't need to force this on the user

File: lua/astrocommunity/git/mini-git/init.lua [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  main = "mini.git",
+}
```
Comment (mehalter) [inline lua/astrocommunity/git/mini-git/init.lua:8]:
```suggestion
  opts = {},
}
```

might as well put this since there is a `setup` function

**Review by mehalter — CHANGES_REQUESTED** [review]

File: lua/astrocommunity/git/mini-git/init.lua [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---@type LazySpec
+return {
+  "echasnovski/mini-git",
+  event = "User AstroFile",
+  cmd = { "Git" },
```
Comment (mehalter) [inline lua/astrocommunity/git/mini-git/init.lua:5]:
```suggestion
  cmd = "Git",
```

unnecessary table

**Review by mehalter — APPROVED** [review]
:shipit:

**Pull request merged**